cumulus_etl/fhir/fhir_utils.py

```diff
@@ -201,10 +201,13 @@ async def get_docref_note(client: FhirClient, docref: dict) -> str:
     if best_attachment_index < 0:
         # We didn't find _any_ of our target text content types.
         # A content type isn't required by the spec with external URLs, so it's possible an unmarked link could be good.
-        # But let's optimistically enforce the need for a content type ourselves by bailing here.
-        # If we find a real-world need to be more permissive, we can change this later.
+        # We've found a real world case where we've found missing data preventing note download; for now,
+        # we are just notifying about it and returning an empty string object.
+        # If we find a actually need this data, we can change this later.
         # But note that if we do, we'll need to handle downloading Binary FHIR objects, in addition to arbitrary URLs.
-        raise ValueError("No textual mimetype found")
+        
+        print(f"Found unexpected mime type '{mimetype}', skipping.")
+        return ""
```

Comment:
the type in question that caused this error was `application/xml` - wanted to talk about this change before commiting to this approach.

Reply:
We can talk about this during our meeting today, but I'm guessing this was a Cerner thing they do - they wrap the doc in a Cerner-specific xml format, with an RDF doc base64-encoded inside. Usually they generate the html version of the doc later in the doc lifecycle, like when it becomes a `status=final` or whatever the status is.

So I'm leery of trying to support `application/xml` - we already (very optimistically) support `application/xhtml+xml`. But arbitrary xml is harder to know what to do with (i.e. how to extract the note text).

Reply:
OK after talking about this in person, I think your beef was more that unhandled mimetypes were stopping `upload-notes` in its tracks, rather than you wanting to handle application/xml. That makes sense.

So I might vote for `upload-notes` catching the ValueError here and printing something sensible there. This method is also used by the ETL and I believe it catches errors and prints the message itself. So that's the established error-processing approach for this anyway.

Reply:
I'm fine with that - is it cool if I keep the erroring mimetype info in the ValueError?

Reply:
Sure - I was having vibes that it might be too low level info, but :shrug: give the people what they crave - mimetypes